charm.go

```diff
@@ -93,7 +94,7 @@ func ComputedSeries(c CharmMeta) []string {
 	seriesSlice := []string(nil)
 	seriesSet := set.NewStrings()
 	for _, base := range manifest.Bases {
-		series := base.String()
+		series := base.Channel.Track
```

Comment:
This will break all the places it is used in Juju. We need utilise a method that gets a series based on the base.

Reply:
Let's chat.  I do not like having series conversion in the charm package.  We've been pulling it into juju. There are also issues with finding new versions etc, we've worked hard in juju to avoid.  Looks like a backwards step.  

We do need a function to convert in juju.

Reply:
As discussed, `ComputedSeries` moves to Juju.